### PR TITLE
Added a test that a user cannot re-use another user’s email address when updating their profile

### PR DESCRIPTION
**Pull request by simong**
At some point we'll want to add in email verification though

### CODE REVIEW

File: node_modules/gh-users/tests/test-users.js [hunk 1]
```diff
@@ -149,5 +149,18 @@ describe('Users', function() {
...
             });
         });
+
+        /**
+         * Test that verifies that a user cannot use another user's email address
+         */
+        it('verify a user cannot use another user\'s email address', function(callback) {
+            TestsUtil.generateTestUsers(global.tests.apps.cam2014, 2, false, function(simon, nico) {
+
+                // Nico shouldn't be able to re-use Simon's email address
```
Comment (nicolaasmatthijs) [inline node_modules/gh-users/tests/test-users.js]: Can we also do a test where a new user is created using an existing email address?

**Comment by nicolaasmatthijs** [comment]
Finished review. Back to @simong for follow-up

File: node_modules/gh-users/tests/test-users.js (hunk 1 above)
Comment (simong) [inline node_modules/gh-users/tests/test-users.js]: Creating users is one of those areas that hasn't received much test love yet somehow ;-)

**Comment by nicolaasmatthijs** [comment]
Looks good. Merged.

**Pull request merged**